packages/webapp/__tests__/PostPage.tsx

```diff
@@ -574,67 +519,3 @@ it('should not cut summary when there is a summary without reaching threshold',
   const fullSummary = await screen.findByText(summaryText);
   expect(fullSummary).toBeInTheDocument();
 });
-
```

Comment:
I see we removed quite a lot of test cases are these all handled elsewhere?

Reply:
Yes, this was just reverted to handle the test cases in its own separate suite.

Reply:
Perfect, then forget about this comment, wanted to verify :D